cvat/apps/events/handlers.py

```diff
@@ -309,7 +317,10 @@ def handle_create(scope, instance, **kwargs):
     payload = _cleanup_fields(obj=payload)
     record_server_event(
         scope=scope,
-        request_id=request_id(),
+        request_info={
+            "id": request_id(),
+            "user_agent": user_agent(),
```

Comment:
You should merge `request_id` and `user_agent` into a single function. Otherwise, using a dictionary doesn't improve anything.